src/scenes/ApplicationRouter/portal/PrescreenResult/index.tsx

```diff
@@ -1,9 +1,10 @@
 import React, { useCallback, useMemo } from 'react';
 
-import { Box, Card, Typography, useTheme } from '@mui/material';
+import { Box, Typography, useTheme } from '@mui/material';
 import { Button } from '@terraware/web-components';
 
 import { Crumb } from 'src/components/BreadCrumbs';
+import Card from 'src/components/common/Card';
```

Comment:
@tommylau523 I think this was causing that layout issue you noticed where it looked like the top of the card was being clipped.